assets/js/blocks/catalog-sorting/block.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "woocommerce/catalog-sorting",
+	"version": "1.0.0",
+	"title": "Catalog Sorting",
+	"description": "Enable customers to change the sorting order of the products.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"align": true,
```

Comment:
I found the `align` support is quite problematic, it uses `float` so we can get this. As I explained in another comment because we need to rely on the `Row` block eventually, what do you think about removing this `align` support?

<img width="1563" alt="image" src="https://user-images.githubusercontent.com/5423135/212444122-4bf03c45-d40c-4d94-9ec8-e4f8837e7d98.png">

Reply:
👍 Removed